APIGOV-27622 New "Manage your runtime security" section (#426)
Axway/amplify-central: content/en/docs/runtime_security/_index.md

---
title: Manage your runtime compliance
linkTitle: Manage your runtime compliance
weight: 495
date: 2024-04-01
hide_readingtime: true
---

> With an increasing number of APIs, API providers and consumers of APIs, there is a pressing need to capture API traffic and scan the API traffic to assess the security threat(s).

Author: ```suggestion
With an increasing number of APIs, API providers and consumers of APIs, there is a pressing need to capture and scan API traffic to assess security threats.
```

> The Amplify API Management platform allows you to assess the runtime compliance of your APIs from a secruity threat perspective. This is known as API Runtime Compliance and has been integrated into the Amplify API Management. Runtime Compliance is an entitlement you can purchase from Axway as part of the API Management platform.

Author: ```suggestion
Amplify's Runtime Compliance entitlement for the API Management platform  allows you to assess the runtime compliance of your APIs from a security threat perspective. 
```

## What is runtime compliance

> Runtime compliance is assessment of the potential security risk based on the real API traffic.  Amplify has integrated with Graylog API Security to monitor and assess the security threats from real captured API traffic.  For example, the captured API traffic is scanned by Graylog API security to detect real attacts, leaks, and other threats to your APIs.

Author: ```suggestion
Runtime compliance is the assessment of potential security risks based on real-time API traffic.  Amplify has integrated with Graylog API Security to monitor and assess API traffic for security threats.  For example, the captured API traffic is scanned by Graylog API security to detect real attacks, leaks, and other threats.
```

> To get started, you have to create an Amplify environment which connects to Graylog API Security and to retrieve the security results.   The Graylog API Security data is correlated with the Managed APIs from the API Gateways connected to the Amplify Enterprise Marketplace to determine a security grading.   For APIs not found as a match to the API Gateways in the Amplify Enterprise Marketplace, these APIs are represented as Runtime Unmanaged APIs with a security grading.  These unmanaged runtime APIs are the group of unknown APIs not registed with an API Gateway. 

Author: ```suggestion
To get started, must create an Amplify environment that connects to Graylog API Security and to retrieve the security results.   The Graylog API Security retrieves the security results and correlates the data with the Managed APIs from the API Gateways connected to the Amplify Enterprise Marketplace to determine a security grading.  

{{< alert title="Note" color="primary" >}} APIs that do not match the API Gateways in the Amplify Enterprise Marketplace are represented as Runtime Unmanaged APIs with a security grading.  These unmanaged runtime APIs are unknown APIs not registered with an API Gateway. See (link) for security grading details.{{< /alert >}} 
```

---
title: Manage your runtime compliance
linkTitle: Manage your runtime compliance
weight: 495
date: 2024-04-01
hide_readingtime: true
---

With an increasing number of APIs, API providers and consumers of APIs, there is a pressing need to capture and scan API traffic to assess security threats.

> Amplify's runtime compliance entitlement for the API Management platform allows you to assess the runtime compliance of your APIs from a security threat perspective. Runtime Compliance is an entitlement you can purchase from Axway as part of the API Management platform.

Reviewer: instead of entitlement i would say "feature" and remove this "for the API Management platform"

Reviewer: Runtime Compliance is an entitlement you can purchase from Axway as part of the API Management platform. 
I would replace this with: Contact Axway Support at support.axway.com or reach out to your Axway representative to learn how to enable the Runtime Compliance for your organization.

Reviewer 2: Edited based on your feedback
